lib/gui/etcher.js

```diff
@@ -39,6 +39,13 @@ electron.app.on('ready', () => {
   // No menu bar
   electron.Menu.setApplicationMenu(null);
 
+  // Make a persistent electron session called 'success-banner'
+  electron.session.fromPartition('persist:success-banner', {
```

Comment:
I thing I said this before, but can we move this to the safe webview component by using `electron.remote.session`?